kratos/python_interface/python_registry.py

```diff
@@ -0,0 +1,370 @@
+import KratosMultiphysics
```

Comment:
please move this file to `python_scripts`

Reply:
We discussed about this at the very beginning. Considering that this is quite internal and that it can be understood as an extension of the c++ registry we (the KTC) decided to place it in here (OFC at the cost of "manually" installing it as you pointed above). Personally, I'm still of the opinion of keeping it like this. However if it is a blocker for you we can discuss it further (@KratosMultiphysics/technical-committee).

Reply:
that makes sense, hadn't considered that!